src/sql/workbench/browser/designer/designer.ts

```diff
@@ -925,8 +964,11 @@ export class Designer extends Disposable implements IThemable {
 							this._propertiesPane.updateDescription(componentDefinition);
 						}
 					}
+					disabledButtons.forEach(b => b.enabled = true);
+				});
+				table.onBlur((e) => {
+					disabledButtons.forEach(b => b.enabled = false);
```

Comment:
if we implement it this way, if i want to move a row up by 2, I have to:
1. select the row
2. click the move up button
3. select the row
4. click the move up button

Am I understanding it correctly? if so, I am afraid that wouldn't work.

Reply:
It would be -
1. select the row
2. click the move up button
3. click the move up botton again (since it's already selected from last select)

Reply:
but onBlur would be triggered already, so the button is disabled,